tests/dummy/app/templates/pie.hbs

```diff
@@ -66,7 +66,7 @@
         <div class="form-group">
           <label class="col-lg-4 control-label">Label Width Multipler</label>
           <div class="col-lg-8">
-            {{scrubber-component min=0 max=0.45 step=0.05 value=labelWidthMultiplier}}
+            {{scrubber-component min="0" max="0.45" step="0.05" value=labelWidthMultiplier}}
```

Comment:
First, to fix compilation error for running `ember s`. All other charts file using string instead of number.